tests/core/test_core.py

```diff
@@ -314,64 +330,12 @@ def test_task_get_template(self):
             assert value == expected_value
             assert [str(m.message) for m in recorder] == [message]
 
-    def test_local_task_job(self):
```

Comment:
These tests are duplicating what was already in test_local_task_job -- we don't need them again.